### PR TITLE
Adding support for non atomic and non CoreOS machines for deploying containerized mons with kv backend

### PR DESCRIPTION
**Pull request by PiotrProkop**

### CODE REVIEW

File: roles/ceph-mon/tasks/docker/start_docker_monitor.yml [hunk 1]
```diff
@@ -102,3 +102,17 @@
...
+  docker:
+    image: "{{ ceph_mon_docker_username }}/{{ ceph_mon_docker_imagename }}"
+    name: "{{ ansible_hostname }}"
+    net: "host"
+    state: "running"
+    privileged: "{{ mon_docker_privileged }}"
+    env: "KV_TYPE={{kv_type}},KV_IP={{kv_endpoint}},MON_IP={{ hostvars[inventory_hostname]['ansible_' + ceph_mon_docker_interface]['ipv4']['address'] }},CEPH_DAEMON=MON,CEPH_PUBLIC_NETWORK={{ ceph_mon_docker_subnet }},{{ ceph_mon_extra_envs }}"
+  when:
+    not is_atomic and
+    ansible_os_family != 'CoreOS'
```
Comment (leseb) [inline roles/ceph-mon/tasks/docker/start_docker_monitor.yml]: can you put the "and" at the end of the line instead?

**Comment by leseb** [comment]
Would you mind expending some of the `when` conditions? So reading is easier :), thanks!

**Comment by leseb** [comment]
Sorry for the pain but can you stick with this format?

```
 when:
  condition and
 condition2
```
Thanjks!

**Comment by leseb** [comment]
up?

**Comment by PiotrProkop** [comment]
OK, done.

**Pull request merged**